packages/devextreme/js/__internal/ui/chat/chat_message_group.ts

```diff
@@ -27,7 +28,7 @@ class MessageGroup extends Widget<MessageGroupOptions> {
   _getDefaultOptions(): MessageGroupOptions {
     return {
       ...super._getDefaultOptions(),
-      messages: [],
+      items: [],
```

Comment:
Why are we renaming this property? If it's require I see that some renaming are missed, for example:

this._renderMessageBubbles(**items**);

_renderMessageBubbles(**messages**): void { }

Reply:
Okay, let's return `messages`

Reply:
It's only a question, and may be were reasons for this change.